src/bunit.web/TestContext.cs

```diff
@@ -73,6 +73,14 @@ public virtual IRenderedComponent<TComponent> Render<TComponent>(RenderFragment
 	public virtual IRenderedFragment Render(RenderFragment renderFragment)
 		=> (IRenderedFragment)this.RenderInsideRenderTree(renderFragment);
 
+	/// <summary>
+	/// Detaches any <see cref="RenderFragment"/> from the active RenderTree. This forces all children to get disposed
+	/// </summary>
+	public void DetachAllComponents()
```

Comment:
```suggestion
	public void DetachComponents()
```

I think I like the name `DetachComponents` better. Its more in line with e.g. `FindComponents` and similar functions, where we do not have the `All` word in there.